shared/src/business/useCases/auth/authenticateUserInteractor.ts

```diff
@@ -8,12 +8,47 @@
  */
 export const authenticateUserInteractor = async (
   applicationContext: IApplicationContext,
-  { code }: { code: string },
+  { code, cognitoLocal }: { code: string; cognitoLocal?: string },
```

Comment:
I think that authenticateUserInteractor probably should not know about cognitoLocal. We should inject in either a local cognito or a real cognito into applicationContext when we start the app so that when we go to use cognito no interactor needs to know whether or not it is using the real one or the local one.

Reply:
Agreed. Moving it to persistence